### DISCUSSION TITLE
Sonar Beacon and Handheld sonar act as invunerable decoys

### DISCUSSION THREAD

**Discussion by BlackMoons** [FakeFishGames/Barotrauma#14510] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Dropping an active sonar beacon or handheld sonar results in enemies attacking it forever and they can't seem to destroy it either. 

### Reproduction steps

Start sub editor, load any ship, grab a diving suit, go outside
Equip sonar beacon or handheld sonar, turn them on
Drop them into the water. return to ship.
Spawn mudraptor near beacons.

They will attack them until the battery runs dry, unable to destroy them.

![image](https://github.com/user-attachments/assets/10b4ea68-2b90-43d5-b826-ecfb2fe1ab0b)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Guh https://github.com/FakeFishGames/Barotrauma/discussions/14428

**Reply by BlackMoons** [reply]
My bad. Didn't see it.
